apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -437,7 +442,13 @@ def render(element_html: str, data: pl.QuestionData) -> str:
     element = lxml.html.fragment_fromstring(element_html)
     answer_name = pl.get_string_attrib(element, "answers-name")
     format = pl.get_enum_attrib(element, "format", FormatType, FormatType.DEFAULT)
-
+    inline = pl.get_boolean_attrib(element, "inline", INLINE_DEFAULT)
+    dropzone_layout = pl.get_enum_attrib(
```

Comment:
Is there a reason this isn't named `solution_placement` instead of `dropzone_layout`?

Reply:
Not sure. That variable name is used elsewhere and predates this PR so I was just continuing to use it to be consistent.

Reply:
Not sure. That variable name was used elsewhere in the code so I just kept using it to be consistent. Happy to change it if you prefer the former.

Reply:
Ya, this variable name is from like 4 or 5 years ago. I don't think anyone here has any particular attachment to it. But We should have consistency across the whole file of course.

Reply:
I'm not really bothered at the moment because the naming is consistent, I'm ok leaving it. If @nwalters512 thinks 'solution_placement' is a much better variable name, then when we change it we should change it everywhere so we still have consistency.

Reply:
I'd encourage you to change this in a fast-follow PR. Even though the variable name is consistent across functions, it's completely inconsistent with the attribute name, which I'd like to remedy.

Reply:
@nwalters512 Agreed and will do! I'll hammer that out this weekend.